### PR TITLE
Ogre v2.2.4

### PR DESCRIPTION
**Pull request by screwt**
Here is a pull request for ogre v2.2.4.
Tested on linux and windows 10. VS 2019
Bullet master branch

Mind the build of bullet on windows it has to be build with the /Md[d] flag
Default build flag in bullet3 cmake script is /Mt[d]

### CODE REVIEW

**Comment by Ybalrid** [comment]
Oh boy, I really need to get cracking on this Ogre 2.2 stuff

File: demo/main.cpp [hunk 1]
```diff
@@ -323,27 +329,28 @@ class BtOgreTestApplication
 		mSDLWindow = SDL_CreateWindow("BtOgre21 Demo", SDL_WINDOWPOS_CENTERED, SDL_WINDOWPOS_CENTERED, width, height, SDL_WINDOW_SHOWN | SDL_WINDOW_RESIZABLE | (fullscreen ? SDL_WINDOW_FULLSCREEN : 0));
 
 		//Get access to native (os/window manager dependent) window information
-		SDL_SysWMinfo wmInfo;
-		SDL_VERSION(&wmInfo.version);
-		SDL_GetWindowWMInfo(mSDLWindow, &wmInfo);
+		SDL_SysWMinfo* wmInfo = new SDL_SysWMinfo;
```
Comment (Ybalrid) [inline demo/main.cpp:332]: Why do you allocate this structure on the heap? It's useless and I don't see a matching `delete`. Please just kep the wmInof as a struct on the stack

Comment (screwt) [reply demo/main.cpp:332]: It's a misstake, how should i proceed?
Update my branch and do another pull request?

Comment (screwt) [reply demo/main.cpp]: Ok i just updated the branch it seems ok to me.

**Comment by Ybalrid** [comment]
this is probably outdated, I am doing some cleaning so I am closing #22 and #23 for now.

I just killed the old AppVeyor script that was not working anymore anyway (used to pull all the binaries from my game engine project website to have ogre and bullet ready to build)

I am not using Ogre these days for anything, so I do not know about the state of things these days

**Pull request closed without merging**